### PR TITLE
Add support for getZoneFile endpoint

### LINKED ISSUE #57: Add support for getZoneFile endpoint
The documentation on the developer site is missing but the fixture is here: https://github.com/dnsimple/dnsimple-developer/blob/master/fixtures/v2/getZoneFile/success.http

### PR DESCRIPTION
**Pull request by jacegu**
Closes #57

### CODE REVIEW

**Comment by jacegu** [comment]
Ready for your 👀

File: lib/dnsimple/zones.ex [hunk 1]
```diff
@@ -49,9 +66,44 @@ defmodule Dnsimple.Zones do
...
+  """
+  @spec get_zone_file(Client.t, String.t | integer, String.t | integer, Keyword.t) :: Response.t
+  def get_zone_file(client, account_id, zone_id, options \\ []) do
+    url = Client.versioned("/#{account_id}/zones/#{zone_id}/file")
+
+    Client.get(client, url, options)
+    |> Response.parse(%{"data" => %Zone.File{}})
+  end
+
+  @spec zone_file(Client.t, String.t | integer, String.t | integer, Keyword.t) :: Response.t
```
Comment (aeden) [inline lib/dnsimple/zones.ex]: Missing docs?

Comment (jacegu) [inline lib/dnsimple/zones.ex]: I've been avoiding adding docs to aliases. Maybe I should simply document them as such?

Comment (aeden) [inline lib/dnsimple/zones.ex]: No, my bad, I thought that was the implementation, but I see the implementation is above, with docs. :shipit:

**Pull request merged**